opengever/apiclient/client.py

```diff
@@ -53,5 +53,22 @@ def get_navigation(self, raw=False):
             )
         return self.session().get(f'{self.url}/@navigation').json()
 
+    def listing(self, raw=False, name='documents', columns=[], **kwargs):
```

Comment:
```suggestion
    def listing(self, name, columns=[], raw=False, **kwargs):
```
1) I'd not have a default listing name; the caller has to decide what kind of content he wants
2) I'd move `raw` to the end because `columns` is more important

Reply:
Thanks! 💚